src/app/data-entries/dropdown/dropdown-sample-3/dropdown-sample-3.component.scss

```diff
@@ -1,8 +1,12 @@
 .button {
     margin: 8px;
-    width: 160px;
+    width: 240px;
 }
 
 .drop-down__scroll-container {
     max-height: 240px;
 }
+
+.igx-drop-down__header igx-icon{
```

Comment:
We should also not target the inner class selector directly. Try using a different selector, for instance:

```scss
igx-drop-down-item[isHeader="true"] igx-icon {
...
}
```